cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,137 @@
+Adding chat to your website is possible with a few easy steps. This solutions spins up an [Amazon API Gateway](https://aws.amazon.com/api-gateway/) endpoint that triggers an [AWS Lambda](https://aws.amazon.com/lambda/) function. This Lambda function invokes the [Amazon Connect](https://aws.amazon.com/connect/) Service [StartChatContact](https://docs.aws.amazon.com/en_pv/connect/latest/APIReference/API_StartChatContact.html) API and returns the result from that call. You could use the below pre built widget to get up and going. This solution is built on top of [Chat JS](https://github.com/amazon-connect/amazon-connect-chat-ui-examples.git). 
 
-## CloudFormation Deployment Steps
+## [Getting started with Pre Built Widget](#Getting-started-with-Pre-Built-Widget)
+## [Getting started with a Custom Widget](#Getting-Started-with-a-Custom-Widget)
 
 ### Pre-requisites
 
-You need an Amazon Connect instance to deploy this [CloudFormation](https://aws.amazon.com/cloudformation/) template. You can use an existing one or create a new one by following our onboarding guide [here](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-get-started.html).
+1. Create an Amazon Connect Instance by signing up [here](https://aws.amazon.com/connect/).
+You need an Amazon Connect instance to deploy the below [CloudFormation](https://aws.amazon.com/cloudformation/) template. You can use an existing one or create a new one by following our onboarding guide [here](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-get-started.html).
 
-If you are using an existing instance, you may need to make a few changes to your instance to enable Chat. Follow the steps [here](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/blob/master/README.md#enabling-chat-in-an-existing-amazon-connect-contact-center) to see what changes you need to make.
+    If you are using an existing instance, you may need to make a few changes to your instance to enable Chat. Follow the steps [here](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/blob/master/README.md#enabling-chat-in-an-existing-amazon-connect-contact-center) to see what changes you need to make.
 
-### Steps
+2. Once you create an Amazon Connect Instance, create a new contact flow, by picking **Sample queue customer** contact flow and **publish it**
```

Comment:
This is hard to follow. If you're going to specify what to do here you need to be very specific.

```markdown
2. After creating your instance, open the Connect console. In the navigation bar, under **Routing**, choose **Contact Flows**. Select **Sample queue customer**.
```

Why do they need to publish it? it should already be published right? Is this where you need to tell them that you want them to copy the ID and the flow ID ?

Reply:
Are they all published to begin with? When you save something it doesn't publish them. 
Are you saying all the default contact flows are already published?

Reply:
yes they are all published by default.

I suggest you run through your own instructions on a fresh connect instance.

Reply:
I did. The first time I ran it I got a weird error that states i Need a contact flow that's valid